api_tests/harness.ts

```diff
@@ -204,39 +205,31 @@ function validTestPath(path: string): boolean {
     );
 }
 
-function expandPath(
-    paths: string[],
-    parentDir: string = "",
-    depth: number = 5
-): string[] {
-    if (!depth) {
-        return [];
-    }
+function expandGlob(paths: string[]): string[] {
```

Comment:
Honestly, I'd vote for removing this "feature" because I think it is quite unexpected. It is not like you are typing it everyday and we have aliases in cargo-make anyway.

Reply:
I think we are better of with less complexity in here compared to a (for now) slightly more inconvenient way of running several tests.
Also, you made it much better already with this PR!

Reply:
This feature is useful to me. I don't think these 10 lines of code will stop anyone to update the code. It's not even that complex.

Reply:
Well, if you insist on it I can't stop you anyway.

I am still surprised that you even took the time to write it given the marginal utility it provides over just using plain glob patterns.